File: 프로그래머스/2/12941. 최솟값 만들기/최솟값 만들기.java
import java.util.Arrays;

class Solution
{
    public int solution(int []A, int []B)
    {
        Arrays.sort(A);
        Arrays.sort(B);
        
        int answer = 0;
        
        for(int i = 0; i < A.length; i++){
            answer = answer + (A[i] * B[B.length - 1 - i]);
        }

        return answer;
    }
}

File: 프로그래머스/2/12941. 최솟값 만들기/최솟값 만들기.js
function solution(A,B){
    var answer = 0;
    
    A.sort((a,b) => a-b);
    B.sort((a,b) => b-a);
    
    for(var i = 0; i<A.length; i ++){
        answer += A[i]*B[i]
    }
    answer
    
    return answer;
}